README.md

```diff
@@ -0,0 +1,18 @@
+# openSUSE and JepFastText image
+
+This project builds on the openSUSE JDK8 image [here](https://github.com/CAFapi/opensuse-java8-images/blob/develop/src/main/docker/jdk/Dockerfile), Python 3, Jep and fastText are then installed on the image.
+Jep is a library used for executing python scripts from within java code, more information on Jep can be found [here](https://github.com/ninia/jep/blob/master/README.rst).
```

Comment:
If this is supposed to be a newline it does not display like that in GitHub so you will need to change the format if that is what you are looking for.

Reply:
It's ok the bit in [] is the display text for the hyperlink and the () is the url